exporter.go

```diff
@@ -282,7 +285,10 @@ For NGINX, the stub_status page must be available through the URI. For NGINX Plu
 func main() {
 	flag.Parse()
 
-	log.Printf("Starting NGINX Prometheus Exporter Version=%v GitCommit=%v", version, gitCommit)
+	fmt.Printf("NGINX Prometheus Exporter Version=%v GitCommit=%v\n", version, gitCommit)
```

Comment:
Could you possibly move the `fmt.Printf` line inside the `if` block and restore the `log.Printf` line after the `if` block? So that we have the following:
```
./nginx-prometheus-exporter -version
NGINX Prometheus Exporter Version=0.8.0 GitCommit=a98f749
```

```
./nginx-prometheus-exporter  -nginx.scrape-uri=http://demo.nginx.com/stub_status
2020/11/02 16:12:48 Starting NGINX Prometheus Exporter Version=0.8.0 GitCommit=a98f749
2020/11/02 16:12:48 Listening on :9113
2020/11/02 16:12:48 NGINX Prometheus Exporter has successfully started
. . .
```

we'd like to keep that log line with `Starting NGINX Prometheus ... ` when the exporter starts normally